### ISSUE TITLE
Upgrade to v9 of the API

### ISSUE THREAD

**Issue by ryancheley** [ryancheley/toggl-to-sqlite#46]
Per an email sent to me by the toggl support staff

> Hey there,
 
> We're reaching out to you because your team has been using an older version of our API ("v8"), to work with the data your team has on Toggl ([toggl.com](https://d2x2mc04.na1.hubspotlinks.com/Ctc/DO+113/d2x2mC04/VX3mBB576b78W6qjN5K1hN3kYV3Z50p5dh0d1N4Yn9N43m2ndW69sMD-6lZ3kDN7p8vflSckWFW2L145t8C3dL4VHHtWQ4QzV4BW1FCZmv6kgm-nVClMlb32TN26Vv9DKT3lzQdmW75cgG68jtRySW1ZBrnS73_jLtVCVP2Q64tWDFW94sx5Y7CJxWmW3_xpjL4R8bVwV5x5tQ64HT1CW4bRg5h90QRDZW3zghDp6VjLqQW1bkWml7wj-jtW819flm7V-cPJN7_0tFycNy0HW2B1wrJ7y2clzW6BTrYK3R8d4FW8WwwYp913V9zf1f_Qgs04)).
 
> If you don't understand what "API" means, please forward this email to a programmer or developer on your team. Here are some links you can share too:
[https://engineering.toggl.com/changes/2022/10/01/api-v8-deprecation/index.html](https://d2x2mc04.na1.hubspotlinks.com/Ctc/DO+113/d2x2mC04/VX3mBB576b78W6qjN5K1hN3kYV3Z50p5dh0d1N4Yn9N-3m2ndW7Y8-PT6lZ3lSW7Ml8Cm2wNDKFW760J5h3R43khW7mNnnL3flZdSW8VnJrb2f8H0pW8p2WKM12hNx1W67S5TC7-N6XFW1B7t6c6B2490W5GYvM17vqygXW5xQ7BQ51m7XZW6K6Fr_12m860W72wmLr71S8rxW5gpgwd87csMHW5NDTht7XdBTSV8LZFJ1pzMZ-W4y-nHm76GVN-W7Pw_ld1LxhV9W7fMDTK5lP_kxW8-BYmF6XLb51N4c0CHV1_kmCVY8Djw6D9Sm8V2V2sC3WJP2kV7KdDm7Rvb8xMNwRgHr8WhsW7n0FZ-5cPKSXW3hgP6520jl2HW9hzhG-3x5r6Bf4cYX-404)
[https://engineering.toggl.com/docs/additional/v8_migration_guide/index.html](https://d2x2mc04.na1.hubspotlinks.com/Ctc/DO+113/d2x2mC04/VX3mBB576b78W6qjN5K1hN3kYV3Z50p5dh0d1N4Yn9N-3m2ndW7Y8-PT6lZ3mPW2tNpk05HDhQxVY3xtk2qrLDrW6XzDl18cgcSqW7yJpGr2LbGqDW5Nbjlw1H6pppW3zwJ_k1Lv7HYN5ss6yzy_bL-W29qW5w2czR0xW7wg3h61qPDpcW6Nq7c88TYnVKW7HHH-Y5qVTHCW67ryL49f90YVW2LsVR46sdHj7W3vg0tf4LXsDnW2CXwgK1Dvhw5W6rnLCh7vN9JvW6-YPN81k8dZxW43bTnd99DRYVN3lcv6rPXxp_W3jHfpl3sS8kWW86hbhC3wHvBLW6b-B1M5kpK72W1fk2k52q8pJkN2YdBWZvBGFvW7Qv3Hv8gsFF7W4Z60ns863KYkf38bWBK04)
To summarize: we're phasing out the old parts of our service because they don't work as well as the newer parts of our service. We'd like you to update how you use our API because:
You'll take advantage of our newer "v9" API, which has more flexibility than "v8".
Starting May 16, we'll start phasing out support for the "v8" API. This may result in decreased reliability.
By May 23, we'll power off our "v8" API, which means all services using it will stop working as well.
Everything you (or your tech team) need to migrate from our "v8" to "v9" API can be found in the links above.
